Configuration/AlCa/python/autoPCL.py

```diff
@@ -1,3 +1,7 @@
+# Important note:
+# due to the limitations of the DBS database schema, as described in
+# https://cms-talk.web.cern.ch/t/alcaprompt-datasets-not-loaded-in-dbs/11146/2,
+# the keys of the dict (i.e. the "PromptCalib*") MUST be shorter than 31 characters
```

Comment:
nice reminder, but if https://github.com/cms-sw/cmssw/pull/38189 gets accepted, if there is such occurrence simply it will crash at runtime

Reply:
True, this is more a future reminder for whoever adds a new wf there :)